lib/livebook_web/live/session_live/app_teams_component.ex

```diff
@@ -0,0 +1,206 @@
+defmodule LivebookWeb.SessionLive.AppTeamsComponent do
+  use LivebookWeb, :live_component
+
+  alias Livebook.Hubs.Provider
+
+  @impl true
+  def update(assigns, socket) do
+    socket = assign(socket, assigns)
+    deployment_groups = Provider.deployment_groups(assigns.hub)
+
+    deployment_group =
+      if assigns.deployment_group_id do
+        Enum.find(deployment_groups, &(&1.id == assigns.deployment_group_id))
+      end
+
+    app_deployment =
+      if deployment_group do
+        Enum.find(deployment_group.app_deployments, &(&1.slug == assigns.app_settings.slug))
+      end
+
+    socket =
+      socket
+      |> assign(
+        settings_valid?: Livebook.Notebook.AppSettings.valid?(socket.assigns.app_settings),
+        app_deployment: app_deployment,
+        deployment_groups: deployment_groups,
+        deployment_group: deployment_group,
+        deployment_group_form: %{"deployment_group_id" => assigns.deployment_group_id},
+        deployment_group_id: assigns.deployment_group_id
+      )
+      |> assign_new(:warnings, fn -> [] end)
+      |> assign_new(:message_kind, fn -> :info end)
+      |> assign_new(:message, fn -> nil end)
```

Comment:
```suggestion
      |> assign_new(:messages, fn -> [] end)
```

Where it is a `[{kind, message}]`.